common-el9.yaml

```diff
@@ -21,11 +21,7 @@ postprocess:
      ln -sr /usr/bin/conmon /usr/libexec/crio/conmon
 
      # Use crun by default
-     sed -i '/\[crio.runtime\]/a default_runtime="crun"' /etc/crio/crio.conf
-     cat >> /etc/crio/crio.conf <<EOF
-     [crio.runtime.runtimes.crun]
-     runtime_path="/usr/bin/crun"
-     EOF
+     sed -i -e '/\[crio.runtime\]/a default_runtime="crun"' -e '/\[crio.runtime.runtimes.crun\]/{n; s/runtime_path = ""/runtime_path = "\/usr\/bin\/crun"/g}' /etc/crio/crio.conf
```

Comment:
we should also drop this. we should not be enabling crun by default on rhel 9 for openshift yet

Reply:
Thanks for the update